Document and change various small things (#534)
discord-userdoccers/discord-userdoccers: pages/gateway/gateway-events.mdx

| ad_personalization_toggles_disabled ^1^   | boolean                                                                                                                    | Whether personalization toggles for quests are disabled                                                                                                                                                                                                                       |
| broadcaster_user_ids **(deprecated)** ^1^ | array[snowflake]                                                                                                           | IDs of users currently broadcasting                                                                                                                                                                                                                                           |

> | regional_feature_config ^1^               | [regional feature config](#regional-feature-config-object) object                                                          | Configuration of features that can be restricted by the region you are connecting from                                                                                                                                                                                        |

Reviewer: ```suggestion
| regional_feature_config ^1^               | [regional feature config](#regional-feature-config-object) object                                                          | Defines which features are enabled or restricted based on the geographic region of the connecting client                                                                                                                                                                                        |
```

###### Gateway Application Structure

| Field          | Type      | Description                                                                     |
| -------------- | --------- | ------------------------------------------------------------------------------- |
| id             | snowflake | The ID of the application                                                       |
| flags          | integer   | The [application's flags](/resources/application#application-flags)             |

> | flags_new      | string    | The [application's flags](/resources/application#application-flags) as a string |

Reviewer: ```suggestion
| flags          | integer   | The [application's flags](/resources/application#application-flags), limited to `1 << 53`             |
| flags_new      | string    | The [application's flags](/resources/application#application-flags), including those surpassing `1<<53`|
```

###### Gateway Application Structure

| Field          | Type      | Description                                                                                             |
| -------------- | --------- | ------------------------------------------------------------------------------------------------------- |
| id             | snowflake | The ID of the application                                                                               |
| flags          | integer   | The [application's flags](/resources/application#application-flags), limited to `1 << 53`               |

> | flags_new      | string    | The [application's flags](/resources/application#application-flags), including those surpassing `1<<53` |

Reviewer: ```suggestion
| flags_new      | string    | The [application's flags](/resources/application#application-flags), including those surpassing `1 << 53` |
```
crazy nit i know 😭

| normal | A normal Gateway session  |
| oauth  | An OAuth2 Gateway session |

###### Regional Feature Config Object

| Field                    | Type   | Description                                                      |
| ------------------------ | ------ | ---------------------------------------------------------------- |

> | age_gated_features       | number | Bitflag of [age gated features](#age-gated-features)             |

Reviewer: ```suggestion
| age_gated_features       | integer | Bitfield representing [age-gated actions and content](#age-gated-features) |
```

> | teen_by_default_settings | number | Bitflag of [teen by default settings](#teen-by-default-settings) |

Reviewer: ```suggestion
| teen_by_default_settings | number | Bitfield representing [safety settings restricted for teens](#teen-by-default-settings) |
```

> ###### Age Gated Features

Reviewer: bit flags should have spaces (`1 << 0` not `1<<0`)

| Value  | Name                                | Description                                         |
| ------ | ----------------------------------- | --------------------------------------------------- |

> | 1\<\<0 | AGE_GATED_SPACES                    | Restricted from guilds marked as NSFW               |

Reviewer: ```suggestion
| 1\<\<0 | AGE_GATED_SPACES                    | Restricted from joining or viewing guilds and channels marked as NSFW               |
```

| 1\<\<1 | SENSITIVE_CONTENT_SHOW_SETTING      | Restricted from changing sensitive content settings |

> | 1\<\<2 | DM_PRIVACY_SETTINGS                 | Restricted from changing DM privacy settings        |

Reviewer: ```suggestion
| 1\<\<2 | DM_PRIVACY_SETTINGS                 | Restricted from changing DM privacy and scanning settings        |
```

> | 1\<\<3 | MESSAGE_REQUEST_RESTRICTIONS_TOGGLE | Restricted from receiving message requests          |

Reviewer: ```suggestion
| 1\<\<3 | MESSAGE_REQUEST_RESTRICTIONS_TOGGLE | Restricted from changing message request filtering settings |
```

> | 1\<\<4 | LARGE_SERVER_ACCESS                 | Restricted from large guilds                        |

Reviewer: ```suggestion
| 1\<\<4 | LARGE_SERVER_ACCESS                 | Restricted from joining or viewing guilds with the [`AGE_VERIFICATION_LARGE_GUILD` feature](/resources/guilds#guild-features)                       |
```

| 1\<\<5 | COMMANDS_TOGGLE                     | Restricted from using commands marked as NSFW       |

> | 1\<\<6 | REACTIVE_CHECK                      | Generic age verification required flag              |

Reviewer: ```suggestion
| 1\<\<6 | REACTIVE_CHECK                      | Requires the client to perform a silent age verification check before accessing gated features             |
```

> | 1\<\<7 | STAGE_SPEAKING                      | Restricted from speaking in stages                  |

Reviewer: ```suggestion
| 1\<\<7 | STAGE_SPEAKING                      | Restricted from starting stages or raising a hand to speak in stage channels |
```

###### Teen By Default Settings

| Value  | Name                                 | Description                                                   |
| ------ | ------------------------------------ | ------------------------------------------------------------- |

> | 1\<\<0 | SENSITIVE_CONTENT                    | Restricts sensitive content settings                          |

Reviewer: ```suggestion
| 1\<\<0 | SENSITIVE_CONTENT                    | Forces sensitive content filters to be enabled |
```

> | 1\<\<1 | FRIEND_REQUEST_STRANGER_CONFIRMATION | Restricts "everyone" option from friend request settings      |

Reviewer: ```suggestion
| 1\<\<1 | FRIEND_REQUEST_STRANGER_CONFIRMATION | Disables the everyone option in friend request settings |
```

| 1\<\<2 | MESSAGE_REQUEST_RESTRICTIONS         | Restricts toggling message requests                           |

> | 1\<\<3 | GUILD_ACTIVITY_STATUS                | Restricts large server options from activity privacy settings |

Reviewer: ```suggestion
| 1\<\<3 | GUILD_ACTIVITY_STATUS                | Forces activity status sharing to be disabled in large guilds |
```

> | 1\<\<4 | SPAM_FILTERS                         | Restricts NSFW content options from spam filter settings      |

Reviewer: ```suggestion
| 1\<\<4 | SPAM_FILTERS                         | Forces spam filters to their highest restriction level in DMs |
```
